src/repositories/user.repository.ts

```diff
@@ -50,4 +50,26 @@ export class UserRepository {
     });
```

Comment:
user.repository.ts
you MUST code more consistent on the design of your artifacts, your sources.
make a new file. products.repositoy.ts in this case. Put your product code there.

Reply:
This was my initial concern I wrote you on the first hand.
Does it make sense to have 2 connections in parallel to the DATABASE?Thanks

Reply:
No it doesn't  have sense,  you must inject the same connection to each repository